插件在solon项目中无法正常工作（The plugin does not work properly in Solon projects.）

### Contact Details

572315466@qq.com

### Version

3.1.2

### Plugin

smart-doc-maven-plugin

### Build tool version

3.9.9

### Jdk version

17

### Describe the bug (Bug描述，社区已开启国际化推广，请用千问、星火等免费大模型工具辅助翻译成英文，减少社区开发者的工作)

插件在solon项目中无法正常工作，生成文档的过程中报如下

The plugin does not work properly in Solon projects and throws the following error during documentation generation

`
[ERROR] 
java.lang.NullPointerException: Cannot invoke "com.ly.doc.model.annotation.RequestPartAnnotation.getAnnotationName()" because the return value of "com.ly.doc.model.annotation.FrameworkAnnotations.getRequestPartAnnotation()" is null
    at com.ly.doc.template.IRestDocTemplate.requestParams (IRestDocTemplate.java:1057)
    at com.ly.doc.template.IRestDocTemplate.buildEntryPointMethod (IRestDocTemplate.java:855)
    at com.ly.doc.template.IRestDocTemplate.processApiData (IRestDocTemplate.java:176)
`

### Expected Behavior (您期望的结果，社区已开启国际化推广，请用千问、星火等免费大模型工具辅助翻译成英文，减少社区开发者的工作)

插件能正常生成接口文档。

The plugin can successfully generate API documentation.

### Current Behavior (当前结果，社区已开启国际化推广，请用文心一言、讯飞星火等辅助翻译成英文，减少社区开发者的工作)

无法正常生成接口文档。

Failed to generate API documentation properly.

### Steps to Reproduce (Bug产生步骤，请尽量提供用例代码。社区已开启国际化推广，请用文心一言、讯飞星火等辅助翻译成英文，减少社区开发者的工作)

1、创建solon项目（版本：3.7.2），引入smart-doc插件
2、运行mvn插件命令

1. Create a Solon project (version: 3.7.2)  and add the smart-doc plugin.  
2. Run the Maven plugin command.

### Possible Solution (Bug解决建议，社区已开启国际化推广，请用文心一言、讯飞星火等辅助翻译成英文，减少社区开发者的工作)

_No response_

### Context (Bug影响描述，社区已开启国际化推广，请用文心一言、讯飞星火等辅助翻译成英文，减少社区开发者的工作)

_No response_

### Validations

- [x] Check if you're on the [latest smart-doc version](https://github.com/smart-doc-group/smart-doc/releases/latest). (请检查是否为最新版本)
- [x] Read the [docs](https://smart-doc-group.github.io/). (请先阅读官方文档)
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate. (检查是否存在报告相同错误的问题，以避免产生重复)

--- a/src/main/java/io/github/smartdoc/template/IRestDocTemplate.java
+++ b/src/main/java/io/github/smartdoc/template/IRestDocTemplate.java
@@ -1054,7 +1054,10 @@ public interface IRestDocTemplate extends IBaseDocBuildTemplate {
 				}
 				if (frameworkAnnotations.getRequestParamAnnotation().getAnnotationName().equals(annotationName)
 						|| frameworkAnnotations.getPathVariableAnnotation().getAnnotationName().equals(annotationName)
-						|| frameworkAnnotations.getRequestPartAnnotation().getAnnotationName().equals(annotationName)) {
+						|| (frameworkAnnotations.getRequestPartAnnotation() != null
+								&& frameworkAnnotations.getRequestPartAnnotation()
+									.getAnnotationName()
+									.equals(annotationName))) {
 					String defaultValueProp = DocAnnotationConstants.DEFAULT_VALUE_PROP;
 					String requiredProp = DocAnnotationConstants.REQUIRED_PROP;
 					// RequestParam annotation
@@ -1072,9 +1075,10 @@ public interface IRestDocTemplate extends IBaseDocBuildTemplate {
 						apiParamEnum = ApiParamEnum.PATH;
 					}
 					// RequestPart annotation
-					else if (frameworkAnnotations.getRequestPartAnnotation()
-						.getAnnotationName()
-						.equals(annotationName)) {
+					else if (frameworkAnnotations.getRequestPartAnnotation() != null
+							&& frameworkAnnotations.getRequestPartAnnotation()
+								.getAnnotationName()
+								.equals(annotationName)) {
 						requiredProp = frameworkAnnotations.getRequestPartAnnotation().getRequiredProp();
 						isRequestPart = true;
 						mockValue = JsonBuildHelper.buildJson(fullyQualifiedName, typeName, Boolean.FALSE, 0,
@@ -1518,7 +1522,10 @@ public interface IRestDocTemplate extends IBaseDocBuildTemplate {
 					paramAdded = true;
 				}
 				// RequestPart annotation
-				else if (frameworkAnnotations.getRequestPartAnnotation().getAnnotationName().contains(annotationName)) {
+				else if (frameworkAnnotations.getRequestPartAnnotation() != null
+						&& frameworkAnnotations.getRequestPartAnnotation()
+							.getAnnotationName()
+							.contains(annotationName)) {
 					if (!JavaClassValidateUtil.isFile(gicTypeName)) {
 						apiMethodDoc.setContentType(MediaType.MULTIPART_FORM_DATA);
 						FormData formData = new FormData();

